panedrlite/panedrlite/panedr.py

```diff
@@ -85,7 +85,8 @@
 Enxnm = collections.namedtuple('Enxnm', 'name unit')
 ENX_VERSION = 5
 
-__all__ = ['edr_to_df', 'edr_to_dict', 'read_edr']
+__all__ = ['ENX_VERSION', 'edr_to_df', 'edr_to_dict', 'read_edr']
```

Comment:
This could maybe use a comment to say that ENX_VERSION is only needed for the tests and not useful for users